centreon/packages/ui/src/Listing/ActionBar/index.tsx

```diff
@@ -79,7 +105,10 @@ const MemoListingActionBar = ({
   onLimitChange,
   moveTablePagination = false,
   widthToMoveTablePagination = 550,
-  actionsBarMemoProps = []
+  actionsBarMemoProps = [],
+  viewerModeData,
+  displayViewerMode,
```

Comment:
```suggestion
  viewerModeConfiguration
```
You can merge those prop and set `viewerModeConfiguration` to undefined in case you don't want to display the viewer mode